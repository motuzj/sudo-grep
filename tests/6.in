search one word + invert match
-v "n"
