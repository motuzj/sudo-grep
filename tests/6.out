Lorem ipsum DoloR sit amet,
No sed do eiusmod tempor
Arcu vitae sed.
Ultrices vitae auctor eu augue ut lectus arcu.
Quis auctor elit sed vulputate mi sit amet.
aasfdasdfasdfsdaF
